tests/performance/src/main/java/org/mule/runtime/core/policy/CompositeOperationPolicyBenchmark.java

```diff
@@ -57,7 +59,7 @@ public Publisher<CoreEvent> apply(Publisher<CoreEvent> publisher) {
       }
     }, "")), empty(), (policy, nextProcessor) -> nextProcessor,
                                            muleContext.getConfiguration().getShutdownTimeout(),
-                                           fluxCompleteScheduler);
+                                           fluxCompleteScheduler, feature -> false);
```

Comment:
Changing to false on this version since the feature flag is not enabled by default in 4.4.x